CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# v1.13.4 (2023-04-24)
+
+## OS Changes
+
+* Ensure the correct hostname is used when a VPC DHCP option set has multiple domains ([#3032])
```

Comment:
It's more that it's the first hostname, right? There's not really a "correct" one.

Reply:
Right - will fix